src/include/sof/samples/audio/c_models_inc/config.h

```diff
@@ -0,0 +1,37 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ *
+ * Copyright 2021 NXP
+ *
+ * Author: Mihai Despotovici <mihai.despotovici@nxp.com>
+ */
+
+#ifndef __CONFIG_H__
+#define __CONFIG_H__
+
+
+// RAW AUDIO DATA CONFIGURATION
```

Comment:
C style comments please.

Reply:
At this moment the model is loaded at build time, but yes, in the future it will be nice to load it at runtime, I agree.

Reply:
@cristinailie ok, runtime means we will need an aux bus kernel "model loader", but I guess @dbaluta already has this in his kernel plans ?

Reply:
@lgirdwood @cristinailie this is on our TODO list for near future (summer of 2021)

Reply:
+ @ujfalusi who will be working on further driver aux bus driver partitioning, need to make sure any aux bus work is generic in nature so we can additionally hook in NN model loading.

Reply:
@lgirdwood I read the thread about aux bus, I'm still yet to grasp how will help us with model loading :). Thanks for pointing this out. @ujfalusi good to hear that you are working on this!